client/my-sites/plans/style.scss

```diff
@@ -12,3 +12,19 @@
 		}
 	}
 }
+
+.plans__trial-copy{
+	position: relative;
+	text-align: center;
+
+	&:after,
+	&:before{
+		content: '';
+	    position: absolute;
+	    top: 50%;
```

Comment:
We indent positions an extra level according to the docs (http://calypso.localhost:3000/devdocs/docs/coding-guidelines/css.md?term=css#positioning)